### ISSUE TITLE
[Discussion] Changing main instance's domain extension

### ISSUE THREAD

**Issue by boehs** [LemmyNet/lemmy#1815]
We have been using `.ml` for a good time now, but I think as lemmy grows it's time to revisit the domain name, for the following reasons:

1. google ranks country domains lower in search results especially for foreigners (to Mali no less)
2. Freenom likes to yank these domains if you forget to renew and force you to pay (in my experience)
3. Freenom is sketchy
4. People might be unlikely to trust free domains
4. And, most importantly, many workplaces and schools (and maybe home firewalls?) block free domain extensions, here is mine doing so

![image](https://user-images.githubusercontent.com/51836263/136078626-35552489-3044-4bca-9d11-fd952bdb2587.png)

With lemmy's growth, I think it's time to reevaluate and point the .ml to an .io or the likes (.io is only 42$ @ gandi.net). If this is financially unfeasible, I would be happy to chip in

**Comment by boehs** [comment]
crap

**Comment by boehs** [comment]
I was here for #263 and I regret not partisipating and mentioning alternate domain names like .io and .org

**Comment by dessalines** [comment]
Domain names add no value whatsoever, and buying them or feels like acquiescing to the digital enclosure of the commons. W/ respect to your specific concerns:

> google ranks country domains lower in search results especially for foreigners (to Mali no less)

Its not really a goal for lemmy.ml to get high in search rankings for google. As @Nutomic said, we'd much rather have an ecosystem of many instances, than a few large ones. People will likely find an instance through https:///join-lemmy.org anyway.

> Freenom likes to yank these domains if you forget to renew and force you to pay (in my experience). Freenom is sketchy.

I've been using them for years w/o problems. If necessary I'll pay the $10 / year, which isn't too big a deal. But a few letters isn't going to make a difference. 
 
> People might be unlikely to trust free domains

We can and should educate them then on why domain names add no value whatsoever, that where and who is hosting your server is infinitely more important.

> And, most importantly, many workplaces and schools (and maybe home firewalls?) block free domain extensions, here is mine doing so

Then people need to start complaining to their administrators who won't let them access any websites associated with certain countries.

---

w/ respect to your 2nd post, we don't want any of this because we don't want a "mainstream", or "flagship" lemmy instance; we want a resilient ecosystem, not a centralized model. We run lemmy.ml in a way that we want to be an enjoyable experience for people.

> Flagships change with time, and if you think that lemmy.ml is not the flagship to be, than I am comfortable moving ship.

Yes, feel free to host your own instance, we encourage this. But no instance will be a "flagship" instance.

**Comment by boehs** [comment]
Excluding the flagship domain (previously was said you were potentially open to it), are you open to research into the other bullets?

**Comment by dessalines** [comment]
Of course, both this and the UI are open source, feel free to contribute and fix issues.
